Manage python type content (#2370)
theforeman/foreman-documentation: guides/common/modules/proc_uploading-files-to-a-python-repository.adoc

[id="Uploading_Files_To_a_Python_Type_Repository_{context}"]
= Uploading Files To a Python Type Repository

> Use this procedure to upload files to a python type repository.

Reviewer: ```suggestion
You can upload local files to a python type repository.
```
or
```suggestion
You can upload files from your local machine to a python type repository.
```

Author: Applied.
